samples/azure/blob-storage-databus/ABSDataBus_5/Receiver/Receiver.csproj

```diff
@@ -6,8 +6,8 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="13.*" />
-    <PackageReference Include="NServiceBus.DataBus.AzureBlobStorage" Version="5.0.0-alpha.119" />
-    <PackageReference Include="NServiceBus" Version="8.0.0-alpha.1896" /> <!-- Remove when AzureBlobStorage package is updated -->
+    <PackageReference Include="NServiceBus.DataBus.AzureBlobStorage" Version="5.0.0-beta.1" />
+    <PackageReference Include="NServiceBus" Version="8.0.0-beta.1" /> <!-- Remove when AzureBlobStorage package is updated -->
```

Comment:
I wasn't sure about these because I lack context on why they are there. There's a bunch of them in snippets and samples and I figured we could update all of them in a separate PR